proxyclient/m1n1/proxyutils.py

```diff
@@ -554,6 +554,13 @@ def bootstrap_port(iface, proxy):
         do_baud = False
 
     if do_baud:
+        # adjust cpufreq for 1500000 as neccessary
```

Comment:
I'd just pull all the logic for checking the chipid into python here and just call proxy.cpufreq_init if required then

Reply:
detecting the chip id in python means pulling the whole adt over 115200 baud which is really slow.

Reply:
you could add a proxy command to request the chipid from m1n1. That's generic and might be useful for other use cases.
Is there a reason why it can't be done on C side in the `P_SET_BAUD` handler?